web/src/main/java/no/nav/modiapersonoversikt/service/utbetaling/UtbetalingUtils.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.modiapersonoversikt.service.utbetaling
+
+import no.nav.modiapersonoversikt.utils.ConvertionUtils.toJavaTime
+import no.nav.tjeneste.virksomhet.utbetaling.v1.informasjon.WSUtbetaling
+import java.time.LocalDate
+import java.util.function.Predicate
+
+object UtbetalingUtils {
+    val EKSTRA_SOKEPERIODE: Long = 20
+
+    @JvmStatic
+    fun finnUtbetalingerISokeperioden(start: LocalDate, slutt: LocalDate) = Predicate<WSUtbetaling> { utbetaling ->
+        val dato = listOfNotNull(
+            utbetaling.utbetalingsdato,
+            utbetaling.forfallsdato,
+            utbetaling.posteringsdato,
+        ).firstOrNull()?.toJavaTime()
+
+        erDatoISokeperioden(dato, start, slutt)
+    }
+
+    @JvmStatic
+    fun leggTilEkstraDagerPaaStartdato(dato: LocalDate): LocalDate {
+        val tidligsteDato = LocalDate.now().minusYears(3).withDayOfYear(1)
+        val nyStartDato = dato.minusDays(EKSTRA_SOKEPERIODE)
+        return if (nyStartDato < tidligsteDato) tidligsteDato else nyStartDato
+    }
```

Comment:
Lurer fortsatt på hva casen rundt denne er, hvorfor må vi gjøre det?

Reply:
Sånn jeg har fortstått det så er det pga hvilke datoer som blir brukt ved filtrering. 
Så en utbetaling kan bli opprettet en god stund i forkant av at selve utbetalingingen skjer, så for å sikre at vi får med oss alt som er relevant utvider vi søkeperioden.